content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -237,24 +255,55 @@ used to verify the serving certificate of the API server.
 Finally, the default namespace to be used for namespaced API operations is placed in a file
 at `/var/run/secrets/kubernetes.io/serviceaccount/namespace` in each container.
 
-From within a Pod, the recommended ways to connect to the Kubernetes API are:
+#### Using kubectl proxy
 
-  - For a Go client, use the official [Go client library](https://github.com/kubernetes/client-go/).
-    The `rest.InClusterConfig()` function handles API host discovery and authentication automatically.
-    See [an example here](https://git.k8s.io/client-go/examples/in-cluster-client-configuration/main.go).
+If you would like to query the API without an official client library, you can run `kubectl proxy`
```

Comment:
How about:
```markdown
If the client library you are using does not support in-cluster authentication…
```

Reply:
This was existing text.  The diff is confusing, please compare on the finished web page:
New: https://deploy-preview-16884--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/access-cluster-api/
vs current: https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/